target/linux/ar71xx/image/legacy-devices.mk

```diff
@@ -427,6 +427,12 @@ define LegacyDevice/dLAN_pro_1200_ac
 endef
 LEGACY_DEVICES += dLAN_pro_1200_ac
 
+define LegacyDevice/DW33D
+  DEVICE_TITLE := DomyWifi DW33D
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ledtrig-usbdev kmod-ath10k
+endef
+LEGACY_DEVICES += DW33D
```

Comment:
Use “mtd -r wirte lede-ar71xx-generic-dw33d-squashfs-sysupgrade.bin firmware" command successfully, but reboot error message：

[   12.540378] VFS: Cannot open root device "(null)" or unknown-block(31,2): error -5
[   12.548048] Please append a correct "root=" boot option; here are the available partitions:
[   12.597613] Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(31,2)
[   12.611010] Rebooting in 1 seconds..